rest/model/distro.go

```diff
@@ -10,6 +12,8 @@ import (
 type APIDistro struct {
 	Name             APIString `json:"name"`
 	UserSpawnAllowed bool      `json:"user_spawn_allowed"`
+	Provider         APIString `json:"provider"`
+	AMI              APIString `json:"ami,omitempty"`
```

Comment:
rename AMI "image_id" and ImageID to be provider agnostic

Reply:
Why would we rename it to image ID if as below you only want to do this branch for the EC2 provider? If we're only returning this for the EC2 provider then we should leave this as AMI.

Reply:
The reason for this is for forwards compatibility. We conceivably may wish to add this in the future for some other Cloud provider, e.g. GCE, where image_id might mean something different. Calling this "image_id" will allow us to re-use this API field when we support another provider.